README.md

```diff
@@ -66,7 +66,11 @@ Chooses what dialect of database you want. Must be `mysql`.
 
 `DATABASE_URL` (no prefix) / `DB_DATABASE_URL` - `string` **required**
 
-Connection string for the database.
+Connection string for the database. 
+
+The connection string requires the following two parameters: `?parseTime=true&multiStatements=true`
+
+Eg: `mysql://user:pass@tcp(127.0.0.1:3306)/database?parseTime=true&multiStatements=true`
```

Comment:
thanks for the PR! adding this sounds good. how about adding this `?parseTime=true&multiStatements=true` in the above example in line 60 (also you don't need `mysql://` there)?
then we can keep the line 71, to explain it's needed.